cluster/service/cluster_service.go

```diff
@@ -248,6 +248,15 @@ func ValidateURL(urlStr *string) error {
 	return nil
 }
 
+// Delete deletes the cluster identified by the given `clusterID`
+func (c clusterService) Delete(ctx context.Context, clusterID uuid.UUID) error {
+	// check that the token belongs to a user
```

Comment:
belongs to the operator you mean? Not a user.

Reply:
fixed in 870041f